### PR TITLE
[WIP] More JSON model work

### PR DESCRIPTION
**Pull request by shadowfacts**
Missed some stuff last time; I'll leave this one open for longer in case I find anything else.

### CODE REVIEW

File: mappings/net/minecraft/client/render/model/ModelRotationContainer.mapping [hunk 1]
```diff
@@ -1,2 +1,3 @@
 CLASS dpe net/minecraft/client/render/model/ModelRotationContainer
 	METHOD b getRotation ()Ldoz;
+	METHOD c isUVLocked ()Z
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/model/ModelRotationContainer.mapping]: Not completely sure of the standard -- should this be `isUvLocked`?

Comment (asiekierka) [reply mappings/net/minecraft/client/render/model/ModelRotationContainer.mapping]: Yes.

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Oh, make sure name static final Gson fields `GSON` instead of `gson`

**Comment by shadowfacts** [comment]
That's not a static field.

**Comment by liach** [comment]
Then everything looks perfect.

**Review by Gegy — APPROVED** [review]

**Pull request merged**